libs/serialization/src/CArchive.cpp

```diff
@@ -236,6 +237,11 @@ CArchive& mrpt::serialization::operator<<(
 {
 	return detail::writeStdVectorToStream(s, a);
 }
+CArchive& mrpt::serialization::operator<<(
+	CArchive& s, const mrpt::aligned_std_vector<float>& a)
+{
+	return detail::writeStdVectorToStream(s, a);
```

Comment:
Do you know if there is any syntax to let the compiler know that a pointer is aligned? something like `align(16) float*`? I ask because using an aligned_std_vector<> will not give that information, in spite of the aligned allocator. Just thinking ahead...

Reply:
Not that i know of. You can build something like gsl not_null<T>.